frontend/src/streamDetails/DetailedStreamStatistics/__tests__/StreamStatisticsPage.test.jsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { beforeEach, describe, expect, it } from "@jest/globals";
+
+import Page from "../../../general/Page";
+import StreamStatisticsPage from "../StreamStatisticsPage";
+import StreamStatisticsPageContents from "../StreamStatisticsPageContents";
+import StreamStatisticsInfo from "../../../model/StreamStatistics/StreamStatisticsInfo";
+import StreamStatsSendInfo from "../../../model/StreamStatistics/StreamStatsSendInfo";
+import StreamStatsReceiveInfo from "../../../model/StreamStatistics/StreamStatsReceiveInfo";
+import StreamStatsLinkInfo from "../../../model/StreamStatistics/StreamStatsLinkInfo";
+import StreamStatsWindowInfo from "../../../model/StreamStatistics/StreamStatsWindowInfo";
+
+import { getSampleStreamStats } from "../../../api/SampleData";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<StreamStatisticsPage/> functional component", () => {
+  let wrapper;
+  const dummyStreamStats = new StreamStatisticsInfo(
+    10,
+    10,
+    new StreamStatsWindowInfo(10, 10, 10),
+    new StreamStatsLinkInfo(10, 10, 10),
+    new StreamStatsSendInfo(10, 10, 10, 10, 10, 10, 10),
+    new StreamStatsReceiveInfo(10, 10, 10, 10, 10, 10, 10, 10, 10)
```

Comment:
would be better to have the values be different so that you can actually make sure that you're reading the correct property

Reply:
5f13db2 👍